src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -207,10 +201,6 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + INVALID_EMAIL_DESC + ADDRESS_DESC_BOB
                 + BIRTHDAY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Email.MESSAGE_CONSTRAINTS);
 
-        // invalid address
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + INVALID_ADDRESS_DESC
-                + BIRTHDAY_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Address.MESSAGE_CONSTRAINTS);
-
```

Comment:
Similarly to above, should this test case remain to continue checking for invalid addresses? On that note I will need to clarify what exactly an invalid address means and whether this invalid address can still be considered invalid if we move to make the address field optional.

Reply:
Yep, similar to above, invalid address is not a concern anymore since all inputs will be accepted as input for address